csaf_2_1/mandatoryTests/mandatoryTest_6_1_42.js

```diff
@@ -0,0 +1,238 @@
+import { PackageURL } from 'packageurl-js'
+import Ajv from 'ajv/dist/jtd.js'
+
+const ajv = new Ajv()
+
+const branchSchema = /** @type {const} */ ({
+  additionalProperties: true,
+  optionalProperties: {
+    branches: {
+      elements: {
+        additionalProperties: true,
+        properties: {},
+      },
+    },
+    product: {
+      additionalProperties: true,
+      optionalProperties: {
+        product_identification_helper: {
+          additionalProperties: true,
+          optionalProperties: {
+            purls: { elements: { type: 'string' } },
+          },
+        },
+      },
+    },
```

Comment:
I'm okay with that explicit definition, but one could refactor and use 
```suggestion
    product: fullProductNameSchema,
```

Correct?